src/yuzu/bootmanager.cpp

```diff
@@ -603,19 +604,34 @@ bool GRenderWindow::LoadOpenGL() {
     auto context = CreateSharedContext();
     auto scope = context->Acquire();
     if (!gladLoadGL()) {
+        QMessageBox::critical(
+            this, tr("Error while initializing OpenGL!"),
+            tr("Your GPU may not support OpenGL, or you do not have the latest graphics driver."));
+        return false;
+    }
+
+    QString renderer = QString::fromUtf8(reinterpret_cast<const char*>(glGetString(GL_RENDERER)));
```

Comment:
```suggestion
    const QString renderer = QString::fromUtf8(reinterpret_cast<const char*>(glGetString(GL_RENDERER)));
```

This should likely be moved closer to its usage as well.

Reply:
Can you clarify what you mean by "This should likely be moved closer to its usage as well"? I use `renderer` both when checking for version 4.3 as well as when unsupported extensions are checked.

Reply:
Ah, didn't see that usage of it. Never mind then.